src/bikeracks/__init__.py

```diff
@@ -42,21 +47,34 @@ def create_app(test_config=None):
         pass
         
     # register close_db and init_db_command with the app
-    from . import db
     db.init_app(app)
     
     # import and register the blueprint from the factory
-    from . import bikes
-    from . import votes
     app.register_blueprint(bikes.bikes)
     app.register_blueprint(votes.votes)
         
     @app.route('/')
     def home(name=None):
         # update map with approved markers
-        # render_template invokes the jinja template engine
         return render_template('base.html', name=name)
         
+    @app.route('/submitFeedback', methods=('POST',))
+    def submitFeedback():
+        csv_file = os.path.join(app.instance_path, "feedback.csv")
+        try:
+            with open(csv_file, 'a') as f:
+                timestamp = int(time())
+                feedback = request.form.get('feedback', '', type=str)
+                row = [timestamp, feedback]
+                # creating a csv writer object 
+                csvwriter = csv.writer(f) 
+                # writing the data rows 
+                csvwriter.writerow(row)
+                return ('OK', 200)
+        except Exception as e:
+            return (e, 500)
+            
```

Comment:
It's weird not to see a return value down here.

I'd put it the return ('OK', 200) at the end of the function rather than at the end of the `try` block.

Reply:
Also, you want to put as little as possible inside the `try` block in general. It makes it easier to understand which parts of the code are risky and easier to read in general. Rearrange this so that the parts that basically can't fail are not inside the `try`.

Reply:
Done.